src/DetailsView/components/left-nav/fluent-side-nav.tsx

```diff
@@ -25,15 +27,20 @@ export class FluentSideNav extends React.Component<FluentSideNavProps> {
         }
         const nav = <DetailsViewLeftNav {...this.props} />;
 
+        const dismisPanel = () => {
```

Comment:
```suggestion
        const dismissPanel = () => {
```
Also in the usage below